<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RzxTrayProp</class>
 <widget class="QWidget" name="RzxTrayProp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBackground" >
     <property name="title" >
      <string>Icon background</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="cbTransparent" >
        <property name="text" >
         <string>Transparent background</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblColor" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QToolButton" name="btnColor" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Select color</string>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>161</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>196</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QGroupBox" name="groupQuickAction" >
     <property name="title" >
      <string>Quick action menus</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="5" column="0" >
       <widget class="QCheckBox" name="cbQuickMail" >
        <property name="text" >
         <string>Send a Mail</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="cbQuickSamba" >
        <property name="text" >
         <string>Open a Samba (Windows share)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="cbQuickHttp" >
        <property name="text" >
         <string>Open a Web Page</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="cbQuickNews" >
        <property name="text" >
         <string>Read News</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbQuickFtp" >
        <property name="text" >
         <string>Open a FTP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbQuickChat" >
        <property name="text" >
         <string>Begin a chat</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QCheckBox" name="cbQuickFav" >
        <property name="text" >
         <string>List Favorites separatedly</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>249</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>cbAutoScale</tabstop>
  <tabstop>sbTraySize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbAutoScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblTraySize</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel" >
     <x>388</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutoScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbTraySize</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>469</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel" >
     <x>469</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTransparent</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblColor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>348</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel" >
     <x>333</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTransparent</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnColor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>426</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
